test/filter.js

```diff
@@ -9,6 +9,13 @@ test('With a where option', t => {
   run('trees', options, 12105, t)
 })
 
+test('With a where option with multiple statements', t => {
+  const options = {
```

Comment:
this is good, could you also add a test that uses the equality operator on strings like in the original issue?